src/MSBuild/XMake.cs

```diff
@@ -4528,6 +4528,15 @@ private static void ShowHelpMessage()
             Console.WriteLine(AssemblyResources.GetString("HelpMessage_4_HelpSwitch"));
             Console.WriteLine(AssemblyResources.GetString("HelpMessage_16_Examples"));
             Console.WriteLine(AssemblyResources.GetString("HelpMessage_37_DocsLink"));
+            Console.WriteLine(AssemblyResources.GetString("HelpMessage_43_GetPropertySwitch"));
+            Console.WriteLine(AssemblyResources.GetString("HelpMessage_44_GetItemSwitch"));
+            Console.WriteLine(AssemblyResources.GetString("HelpMessage_45_GetTargetResultSwitch"));
+            Console.WriteLine(AssemblyResources.GetString("HelpMessage_51_GetResultOutputFileSwitch"));
+            Console.WriteLine(AssemblyResources.GetString("HelpMessage_46_FeatureAvailabilitySwitch"));
+            Console.WriteLine(AssemblyResources.GetString("HelpMessage_47_TerminalLoggerSwitch"));
+            Console.WriteLine(AssemblyResources.GetString("HelpMessage_48_TerminalLoggerParametersSwitch"));
+            Console.WriteLine(AssemblyResources.GetString("HelpMessage_49_WaitForDebuggerSwitch"));
```

Comment:
To be fair, the corresponding `ParameterlessSwitchInfo` is ifdefed out in release builds so it will print only when running debug MSBuild. Still probably not worth keeping the resource string and localizing it.

Reply:
removed the -waitfordebugger helpmessage